thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/onboard/DetectionOnboardExecutionContext.java

```diff
@@ -0,0 +1,16 @@
+package com.linkedin.thirdeye.anomaly.onboard;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class DetectionOnboardExecutionContext {
```

Comment:
Please provide a java doc describing the the context here.

Reply:
Added.